service/dap/server.go

```diff
@@ -847,6 +873,12 @@ func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
 			return
 		}
 		program = debugbinary
+
+		if mode == "test" && args.Cwd == "" {
+			// In test mode, run the test binary from the package directory
+			// like in `go test` and `dlv test` by default.
+			args.Cwd = getPackageDir(args.Program, buildDir)
```

Comment:
Why would the fallback be buildDir and not server's cwd?
This is part of running, not part of building.

Reply:
Hoping that user specified the package directory or file directory as buildDir, so even when `go list` fails for some reason, that can be a closer approximation to how users would run `dlv test`.

Reply:
That's confusing.
Do we expect users to use buildDir or the client?

Reply:
Now fallback to . (the delve's working directory)
Most users shouldn't set delveCwd.